### PR TITLE
Orchestration metadata and states

### PR DESCRIPTION
**Pull request by atengler**

### CODE REVIEW

File: glusterfs/orchestrate/server.sls [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    - sls: glusterfs.server.service
+    - batch: 1
+
+glusterfs.server.setup:
+  salt.state:
+    - tgt: 'glusterfs:server'
+    - tgt_type: pillar
+    - queue: True
+    - sls: glusterfs.server.setup
+    - batch: 1
```
Comment (fpytloun) [inline glusterfs/orchestrate/server.sls:15]: You can run this only on one node, no need to run on all of them.

File: glusterfs/orchestrate/server.sls [hunk 2]
```diff
@@ -0,0 +1,18 @@
+glusterfs.server.service:
+  salt.state:
+    - tgt: 'glusterfs:server'
+    - tgt_type: pillar
+    - queue: True
+    - sls: glusterfs.server.service
+    - batch: 1
```
Comment (fpytloun) [inline glusterfs/orchestrate/server.sls:7]: You can run this (glusterfs.server.service) on all nodes at once.

File: glusterfs/orchestrate/server.sls (hunk 1 above)
Comment (atengler) [reply glusterfs/orchestrate/server.sls:15]: There is no way how to run this state only on first matched minion

File: glusterfs/orchestrate/server.sls (hunk 2 above)
Comment (atengler) [reply glusterfs/orchestrate/server.sls:7]: After a lot of time spent on testing Salts behaviour during orchestrate run, I prefer batch run over concurrent run

Comment (fpytloun) [reply glusterfs/orchestrate/server.sls:7]: Using batch run when it's not necessary will result in much slower deployment/updates.
And in this case it's really not needed.

**Pull request merged**